{
  "end": [
    null,
    15329520935541482,
    109952946870224
  ],
  "start": [
    "2018-07-28",
    15329520935541055,
    109952946870224
  ],
  "status": [
    "Archived",
    15329520935539071,
    109952946870224
  ],
  "text": [
    [
      15329520935540390,
      109952946870224,
      "Archive issue after the one closed on github"
    ]
  ],
  "text.trace": [
    "Archive issue after the one closed on github"
  ],
  "tracked": {
    "external_id": "82",
    "provider": "github",
    "source": "ff-notes/ff",
    "url": "https://github.com/ff-notes/ff/pull/82"
  }
}
